iml-api/src/graphql.rs

```diff
@@ -344,7 +344,7 @@ impl QueryRoot {
         msg: Option<String>,
     ) -> juniper::FieldResult<Vec<Command>> {
         let dir = dir.unwrap_or_default();
-        let is_completed = !is_active.unwrap_or(false);
+        let is_completed = is_active.map(|active| !active);
```

Comment:
```suggestion
        let is_completed = is_active.unwrap_or_default();
```

Reply:
Both variants aren't obvious :)

Reply:
1. Does it make sense to rename the argument to `is_complete` then to reduce the "impedance mismatch" or leave it `is_active`? What is more natural/frequent use case?

2. Also having it optional makes us to completely ignore the filter, so
- `commands(isComplete: true)` returns completed commands
- `commands(isComplete: false)` returns incomplete commands
- `commands()` returns all commands
It sounds as an unnecessary feature, right?

Reply:
I think the most common case will be looking for active commands

Reply:
![Screenshot 2020-09-23 at 08 43 27](https://user-images.githubusercontent.com/940363/93958800-1dbcbd00-fd79-11ea-8809-01cc0d21adf7.png)